### PR TITLE
Update configure-advanced-delivery.md

### PR DESCRIPTION
**Pull request by ssaini-pm**
Adding details on how to view messages due to the new advanced delivery system overrides to help admins navigate experiences. Adding clarification for common questions received from customers.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 1]
```diff
@@ -39,22 +39,23 @@ You use the _advanced delivery policy_ in Microsoft 365 to prevent these message
 
 - Filters in EOP and Microsoft Defender for Office 365 take no action on these messages.<sup>\*</sup>
 - [Zero-hour Purge (ZAP)](zero-hour-auto-purge.md) for spam and phishing take no action on these messages.<sup>\*</sup>
-- [Default system alerts](alerts.md) aren't triggered for these scenarios.
+- [Default system alerts](https://docs.microsoft.com/en-us/microsoft-365/compliance/alert-policies?view=o365-worldwide#default-alert-policies) aren't triggered for these scenarios.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
- [Default system alerts](/microsoft-365/compliance/alert-policies?view=o365-worldwide#default-alert-policies) aren't triggered for these scenarios.
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 2]
```diff
@@ -39,22 +39,23 @@ You use the _advanced delivery policy_ in Microsoft 365 to prevent these message
...
+- [Default system alerts](/microsoft-365/compliance/alert-policies?view=o365-worldwide#default-alert-policies) aren't triggered for these scenarios.
 - [AIR and clustering in Defender for Office 365](office-365-air.md) ignores these messages.
 - Specifically for third-party phishing simulations:
-  - [Admin submissions](admin-submission.md) generates an automatic response saying that the message is part of a phishing simulation campaign and isn't a real threat. Alerts and AIR will not be triggered.
+  - [Admin submissions](admin-submission.md) generates an automatic response saying that the message is part of a phishing simulation campaign and isn't a real threat. Alerts and AIR will not be triggered. The admin submissions experience will show these messages as a simulated threat.
+  - When a user reports a phish simulation message using the [Report Phishing add-in Outlook](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/enable-the-report-message-add-in?view=o365-worldwide), system will not generate alert, investigation or incident. Message will also show up on the User reported messages tab of the submissions page.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
  - When a user reports a phish simulation message using the [Report Phishing add-in for Outlook](/microsoft-365/security/office-365-security/enable-the-report-message-add-in?view=o365-worldwide), the system will not generate an alert, investigation, or incident. The message will also show up on the User reported messages tab of the submissions page.
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 3]
```diff
@@ -39,22 +39,23 @@ You use the _advanced delivery policy_ in Microsoft 365 to prevent these message
...
   - [Safe Links in Defender for Office 365](safe-links.md) doesn't block or detonate the specifically identified URLs in these messages.
   - [Safe Attachments in Defender for Office 365](safe-attachments.md) doesn't detonate attachments in these messages.
 
 <sup>\*</sup> You can't bypass malware filtering or ZAP for malware.
 
-Messages that are identified by the advanced delivery policy aren't security threats, so the messages are marked as system overrides. Admins can filter and analyze these system overrides in the following experiences:
+Messages that are identified by the advanced delivery policy aren't security threats, so the messages are marked with system overrides. Admin experiences will show these messages as due to either a "phishing simulation" system override or a "SecOps mailbox" system override. Admins can filter and analyze on these system overrides in the following experiences:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md:52]:
```suggestion
Messages that are identified by the advanced delivery policy aren't security threats, so the messages are marked with system overrides. Admin experiences will show these messages as due to either a "phishing simulation" system override or a "SecOps mailbox" system override. Admins can filter and analyze these system overrides in the following experiences:
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 4]
```diff
@@ -39,22 +39,23 @@ You use the _advanced delivery policy_ in Microsoft 365 to prevent these message
...
 
-- [Threat Explorer/Real-time detections in Defender for Office 365 plan 2](threat-explorer.md)
-- The [Email entity Page in Threat Explorer/Real-time detections](mdo-email-entity-page.md)
-- The [Threat protection status report](view-email-security-reports.md#threat-protection-status-report)
-- [Advanced hunting in Microsoft Defender for Endpoint](../defender-endpoint/advanced-hunting-overview.md)
-- [Campaign Views](campaigns.md)
+- [Threat Explorer/Real-time detections in Defender for Office 365 plan 2](threat-explorer.md): Admin can filter on "System override source" and select either "Phishing Simulation" or "SecOps Mailbox."
+- The [Email entity Page in Threat Explorer/Real-time detections](mdo-email-entity-page.md): Admin can view a message was allowed by organization policy by either "SecOps mailbox" or "Phishing Simulation" under Tenant override in the Override(s) section.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
- The [Email entity Page in Threat Explorer/Real-time detections](mdo-email-entity-page.md): Admin can view a message that was allowed by organization policy by either "SecOps mailbox" or "Phishing Simulation" under Tenant override in the Override(s) section.
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 5]
```diff
@@ -39,22 +39,23 @@ You use the _advanced delivery policy_ in Microsoft 365 to prevent these message
...
+- The [Threat protection status report](view-email-security-reports.md#threat-protection-status-report): Admin can filter by "view data by System override" dropdown menu and select to see messages allowed due to a Phishing Simulation system override. To see messages allowed by the SecOps mailbox override, admin can select "chart breakdown by delivery location" under "chart breakdown by reason" dropdown menu.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
- The [Threat protection status report](view-email-security-reports.md#threat-protection-status-report): Admin can filter by "view data by System override" dropdown menu and select to see messages allowed due to a Phishing Simulation system override. To see messages allowed by the SecOps mailbox override, the admin can select "chart breakdown by delivery location" under the "chart breakdown by reason" dropdown menu.
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 6]
```diff
@@ -99,7 +100,7 @@ The SecOps mailbox entries that you configured are displayed on the **SecOps mai
    - Click ![Edit icon](../../media/m365-cc-sc-edit-icon.png) **Edit**.
    - If there are no configured phishing simulations, click **Add**.
 
-3. On the **Edit third-party phishing simulation** flyout that opens, configure the following settings:
+3. On the **Edit third-party phishing simulation** flyout that opens, configure the following settings: 
 
    - **Sending domain**: Expand this setting and enter at least one email address domain (for example, contoso.com) by clicking in the box, entering a value, and then pressing Enter or selecting the value that's displayed below the box. Repeat this step as many times as necessary. You can add up to 10 entries.
    - **Sending IP**: Expand this setting and enter at least one valid IPv4 address is required by clicking in the box, entering a value, and then pressing Enter or selecting the value that's displayed below the box. Repeat this step as many times as necessary. You can add up to 10 entries. Valid values are:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
   - **Sending IP**: Expand this setting and enter at least one valid IPv4 address by clicking in the box, entering a value, and then pressing Enter or selecting the value that's displayed below the box. Repeat this step as many times as necessary. You can add up to 10 entries. Valid values are:
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 7]
```diff
@@ -109,6 +110,8 @@ The SecOps mailbox entries that you configured are displayed on the **SecOps mai
    - **Simulation URLs to allow**: Expand this setting and optionally enter specific URLs that are part of your phishing simulation campaign that should not be blocked or detonated by clicking in the box, entering a value, and then pressing Enter or selecting the value that's displayed below the box. You can add up to 10 entries. For the URL syntax format, see [URL syntax for the Tenant Allow/Block List](/microsoft-365/security/office-365-security/tenant-allow-block-list#url-syntax-for-the-tenant-allowblock-list).
 
    To remove an existing value, click remove ![Remove icon](../../media/m365-cc-sc-remove-selection-icon.png) next to the value.
+   
+   Note: You must specify at least one Sending Domain and at least one Sending IP to configure a third-party phishing simulation in Advanced Delivery. You may optionally include Simulation URLs to allow to ensure URLs present in simulation messages are not blocked. You may specify up to 10 entries for each field. There must be a match on at least one Sending Domain and one Sending IP but no association between values is maintained.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
   > [!NOTE]
   > You must specify at least one Sending Domain and at least one Sending IP to configure a third-party phishing simulation in Advanced Delivery. You may optionally include Simulation URLs to allow to ensure URLs present in simulation messages are not blocked. You may specify up to 10 entries for each field. There must be a match on at least one Sending Domain and one Sending IP but no association between values is maintained.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ssaini-pm, kindly commit the suggested changes. Many thanks.

**Pull request merged**